app/helpers/content_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ContentHelper
+  def split_tag(value, delimiter)
+    if value.is_a? Array
+      return value unless value.join.include? delimiter
+      value.flat_map { |str| str.partition(delimiter) }.reject(&:empty?)
```

Comment:
Looks like this is not needed. I simplified this to `return value` and all tests passed. This is because the array never includes the delimiter because the array has escaped quotes, but the delimiter does not, which doesn't matter because it works without having to do any further processing.

Alternatively, we might want to separate the different components of the sentence into different i18n entries and safe_join them without having to create this helper method, which took me some time to figure out what it was doing and why.

Reply:
Were you able to confirm that this line is not needed? It looks like if the value is an Array, we can always return the value.

Reply:
The way it stands it is needed. If we were to use your alternate suggestion, it wouldn't be

> Alternatively, we might want to separate the different components of the sentence into different i18n entries and safe_join them without having to create this helper method, which took me some time to figure out what it was doing and why.

I'm not sure which one you view as cleaner. I think they are both messy